libnd4j/include/ops/declarable/generic/random/uniform.cpp

```diff
@@ -35,41 +36,40 @@ namespace nd4j {
          * TArgs[0] - min for rng
          * TArgs[1] - max for rng
          */
-        CUSTOM_OP_IMPL(randomuniform, 1, 1, true, 2, 0) {
+        CUSTOM_OP_IMPL(randomuniform, 3, 1, true, 0, 0) {
```

Comment:
No. revert it back here. min/max arrays must be optional, as well as INT_ARGS/T_ARGS

Reply:
Yes, changed to use min/max vals as optional with default min = 0, max = +inf